packages/popmotion/src/_tests/calc.test.ts

```diff
@@ -101,4 +101,13 @@ describe('stepProgress', () => {
     expect(stepProgress(3, .9)).toBe(1);
     expect(stepProgress(3, .1)).toBe(0);
   });
+  
+  it('should return the correct progress 0 - 1 on border boundaries', () => {
```

Comment:
hm, is this correct behaviour? non edge segments are two times wider than edge ones, shouldnt distribution be even?

Reply:
My assumption was that each step would have an equal deviation from which it would snap to that step. kinda like a sinusoid where each step is a valley, once you get to a peak (odd numbered subsection) you'd go into the next valley. That assumption could be wrong tho.

My use case was achieving a scroll to this: http://www.benmingo.com/ . Would basically tell me the current project based on percentage of a container thats been scrolled.

Do you see it being used differently?